src/modules/0.0.17/aws_account/index.ts

```diff
@@ -30,20 +50,12 @@ class AwsAccount extends ModuleBase {
     },
     awsClient: null, // Just reserving this name to guard against collisions between modules.
   };
-  awsAccount = new Mapper2<AwsAccountEntity>({
-    entity: AwsAccountEntity,
-    equals: (_a: AwsAccountEntity, _b: AwsAccountEntity) => true,
-    source: 'db',
-    cloud: new Crud2({
-      create: async (_e: AwsAccountEntity[], _ctx: Context) => { /* Do nothing */ },
-      read: (ctx: Context, id?: string) => ctx.orm.find(AwsAccountEntity, id ? {
-        where: {
-          id,
-        },
-      } : undefined),
-      update: async (_e: AwsAccountEntity[], _ctx: Context) => { /* Do nothing */ },
-      delete: async (_e: AwsAccountEntity[], _ctx: Context) => { /* Do nothing */ },
-    }),
-  });
+  awsAccount: AccountMapper;
```

Comment:
why is this not `mapper: AccountMapper`? wouldn't that be more standardized?

Reply:
A module can have multiple mappers (and usually does). This module is an exception in that way, too, but I wanted to minimize the number of differences this one has versus other modules, if possible.